static/js/stream_popover.js

```diff
@@ -500,6 +533,27 @@ exports.register_topic_handlers = function () {
 
         e.stopPropagation();
     });
+
+    $('body').on('click', '.sidebar-popover-move-topic-messages', function (e) {
+        const topic_row = $(e.currentTarget);
+        const stream_id = topic_row.data('stream-id');
+        const topic_name = topic_row.data('topic-name');
```

Comment:
We generally access via `.attr('data-stream-id')` as a codebase convention, because using jQuery's `.data()` to write doesn't update the attribute in the DOM, which can be super confusing.